mix.exs

```diff
@@ -5,8 +5,8 @@ defmodule ElixirBoilerplate.Mixfile do
     [
       app: :elixir_boilerplate,
       version: "0.0.1",
-      elixir: "~> 1.12",
-      erlang: "~> 24.1",
+      erlang: "~> 25.0",
+      elixir: "~> 1.13",
```

Comment:
Is this also related to Dialyzer?

Reply:
It's a leftover from #166, it looks like I forgot to upgrade the specs in `mix.exs`